docs/fabric/downloads.md

```diff
@@ -2,4 +2,4 @@ The reports contained in this site are available for download in pdf format from
 
 | Fabric Version | Smart Contract | SDK Client | Link |
 | --------- | --------------- | --------------- | ---------------- |
-| 1.4.0 | JavaScript | NodeJS | <a href="../../../pdf/fabric/1.4.0/Fabric_1.4.0_javascript_node.pdf" target="_blank">pdf report</a> |
\ No newline at end of file
+| 1.4.0 | JavaScript | NodeJS | <a href="../resources/pdf/1.4.0/Fabric_1.4.0_javascript_node.pdf" target="_blank">pdf report</a> |
```

Comment:
downloads.md file and resources directory are both in the fabric directory, so should be href="./resources/pdf/1.4.0/Fabric_1.4.0_javascript_node.pdf"

Reply:
I thought that too, but doesn't work locally with the single `.` :/